docs/io/fits/usage/table.rst

```diff
@@ -365,3 +365,130 @@ to the HDU's data attribute:
 
 In a future version of Astropy table creation will be simplified and this
 process won't be necessary.
+
```

Comment:
The lines just above your new stuff is an example where we promise something for "a future version of astropy".